.circleci/workflow-queue-v1.sh

```diff
@@ -14,6 +14,11 @@ BRANCH="master"
 WORKFLOW_NAME="build-test-deploy"
 PROJECT_SLUG="all-of-us/workbench"
 
+# List of jobs in build-test-deploy workflow on master branch.
```

Comment:
I'm a bit worried about keeping this list in sync. Can you just list the specific job this script needs to block on? i.e. `puppeteer-test-2-1` ?Can we just ignore the rest?

Reply:
Good point. The list can be shorter, include downstream jobs only.  
```
JOB_LIST=("puppeteer-test-2-1" "ui-deploy-to-test" "api-deploy-to-test")
```

Also, I cleaned up the script, it's much shorter now. 

I am NOT against idea of rewriting in python or ruby. It will just take development time. I believe updated script that will work better than before. It's not worth of rewrite, right?